example/retarget.csolution.yml

```diff
@@ -1,12 +1,14 @@
 solution:
+  created-for: CMSIS-Toolbox@2.0.0
+  cdefault:
 
   packs:
     - pack: ARM::CMSIS
     - pack: ARM::CMSIS-Compiler
```

Comment:
CMSIS and CMSIS-Compiler pack are already specified in cproject.

Reply:
Will be removed